tests/data_protection/oadp/conftest.py

```diff
@@ -119,6 +120,7 @@ def rhel_vm_with_data_volume_template(
             vm=vm,
             filename=FILE_NAME_FOR_BACKUP,
             content=TEXT_TO_TEST,
+            console_impl=console.RHEL,
```

Comment:
This was removed recently, please check if you need this

Reply:
+1. Please always ensure you have updated your fork

Reply:
ack